server/graphql_schemas/attend/schema.py

```diff
@@ -0,0 +1,65 @@
+import graphene
+from api.models import Attend, Event, GoogleUser
+from graphene import relay, ObjectType
+from graphene_django.filter import DjangoFilterConnectionField
+from graphene_django.types import DjangoObjectType
+from graphql import GraphQLError
```

Comment:
From https://www.python.org/dev/peps/pep-0008/#imports:

Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.